### PR TITLE
Replace `unmangle` with `cygpath`

### PR DESCRIPTION
**Pull request by Rangi42**
Thanks to ISSOtm for first using cygpath in the rgbasm tests (#1472).

### CODE REVIEW

**Review by ISSOtm — APPROVED** [review]
Looks good to me! Just need @Rangi42's approval of the changes I just introduced&mdash;I'll let her merge the PR if she agrees.

**Comment by Rangi42** [comment]
I'd like to understand first why `touch`ing the .exes fixes this.

**Pull request merged**
